.github/workflows/ci.yml

```diff
@@ -215,7 +215,7 @@ jobs:
       - name: "Build"
         id: "build"
         run: |
-          for package in '.#packages.${{ matrix.system }}.flox-pkgdb' '.#packages.${{ matrix.system }}.flox-cli' '.#packages.${{ matrix.system }}.flox' ; do
+          for package in '.#packages.${{ matrix.system }}.flox-pkgdb' '.#packages.${{ matrix.system }}.flox-cli' '.#packages.${{ matrix.system }}.flox' '.#packages.${{ matrix.system }}.flox-cli-tests'; do
```

Comment:
Is everything built here getting copied to the substituter? I don't quite follow how

Reply:
[flox/configure-nix-action](https://github.com/flox/flox/blob/main/.github/actions/common-setup/action.yml#L43-L52) action setups the post action that uploads to the binary cache.

`flox/configure-nix-action@main` is part of the [Setup](https://github.com/flox/flox/pull/2060/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR201-R213) step.

Reply:
Ah makes sense, thanks!